### ISSUE TITLE
Valkyria Chronicles 4 colors issue

### ISSUE THREAD

**Issue by fuhuan416** [yuzu-emu/yuzu#5783]
EA-1349
I am using the Hong Kong version of the game. The game status is playable, and there are some minor color errors. The following is a comparison chart. The picture above is Yuzu, and the picture below is Switch.
![0100c11009378000_2021-01-21_19-48-49-759](https://user-images.githubusercontent.com/6281757/105347913-867e4e00-5c22-11eb-82c4-51f51ed030ca.png)
![1](https://user-images.githubusercontent.com/6281757/105347884-7b2b2280-5c22-11eb-9333-628d67ae311c.jpg)
The above is Intro, it looks like CG animation, not real-time rendering
What needs to be added is that this color problem also occurs after entering the game.

![0100c11009378000_2021-01-21_19-58-44-709](https://user-images.githubusercontent.com/6281757/105348455-56837a80-5c23-11eb-8fb0-e705790c5273.png)
![3436324-valkyria chronicles 9](https://user-images.githubusercontent.com/6281757/105348464-58e5d480-5c23-11eb-8364-a526116f8fd9.jpg)

This is a comparison between the two games. The picture above is Yuzu, and the picture below is Switch
[yuzu_log - Valkyria Chronicles 4.zip](https://github.com/yuzu-emu/yuzu/files/5849111/yuzu_log.-.Valkyria.Chronicles.4.zip)

**Comment by fuhuan416** [comment]
And the textures of the grounds
![0100c11009378000_2021-01-21_20-08-02-342](https://user-images.githubusercontent.com/6281757/105349397-9ac34a80-5c24-11eb-9d64-1451f6ac9344.png)

**Comment by fuhuan416** [comment]
EA-1368
This time I observed a change. After the emulator loads the game, the first screen of the game is the Loading interface. At this time, the color of the Loding picture is still normal. After a few seconds of loading, the Loading picture changes color and becomes the top That kind of color. At this time, I paused the emulator at the fastest speed and backed up the Log file.
[yuzu_log - Valkyria Chronicles 4.zip](https://github.com/yuzu-emu/yuzu/files/5862747/yuzu_log.-.Valkyria.Chronicles.4.zip)
